newsletter/09.09.2020_newsletter.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Septemnber 9, 2020 Newsletter
+permalink: /newsletter09092020/
+layout: page
+exclude: true
+---
+*This is a periodic newsletter of the interesting things we’ve seen and what we are thinking about in open source policy analysis.*
+
+**Analysis of business investment under Biden tax policy plan with
+Cost-of-Capital Calculator and Tax-Calculator.** In a new report, Kyle
+Pomerleau (AEI) used Tax-Calculator and Cost-of-Capital-Calculator to
+analyze the tax burden on business investment that would stem from Joe
+Biden’s proposed tax policy. Results show the marginal effective tax
+rate (METR) for business investment would be 7.8 percentage points
+higher over the next year relative to current law and that the proposed
+tax legislation would favor debt-financed and noncorporate investment.
+Alongside the report, Pomerleau partnered with Matt Jensen (AEI) and
+Peter Metz (AEI) to develop a web app visualizing METRs on capital under
+current tax law and Biden’s proposed tax law.
+[Paper](https://www.aei.org/wp-content/uploads/2020/09/The-tax-burden-on-business-investment-under-Joe-Bidens-tax-proposals.pdf),
+[Visualization](https://t.co/oI11965KSB?amp=1)
```

Comment:
@chusloj This URL looks funky. Want to change to https://compute.studio/AEIEconomics/ccc-biden/viz/ to avoid redirects?

Reply:
@Peter-Metz Resolved.